src/App.js

```diff
@@ -1,25 +1,38 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from "react-router-dom";
+import "./App.css";
+import Home from "./components/Home";
+import AllBeers from "./components/AllBeers";
+import RandomBeer from "./components/RandomBeer";
+import NewBeer from "./components/NewBeer";
+import BeerDetails from "./components/BeerDetails";
+import { useState, useEffect } from "react";
+import axios from "axios";
 
 function App() {
+  const [beers, setBeers] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://ih-beers-api2.herokuapp.com/beers")
```

Comment:
For your project, it is great practice to have route saved in environment variables, like this
```
 axios.get(`${process.env.REACT_APP_API_BASE_URL}/beers`)
```
and accordingly in your .env file

```
REACT_APP_API_BASE_URL=https://ih-beers-api2.herokuapp.com
```